setup.py

```diff
@@ -316,6 +317,10 @@ def main():
             "easygui",
             'colorclass',
         ],
+        extras_require = {
```

Comment:
Christian, is there a specific reason to put it as an optional dependency? I think it would be simpler to make it mandatory, to avoid several checks, specific exceptions, etc. What do you think?

Reply:
msoffcrypto-tools depends on cryptography and others, so it would introduce a rather long tail of other dependencies. But if you do not mind we can make it a hard dependency. That would simplify the code a bit and leave fewer scenarios for testing.